### ISSUE TITLE
Support entering idea without opening editor

### ISSUE THREAD

**Issue by simeg** [simeg/eureka#1]
Like `git commit` vs. `git commit -m [msg]`. Sometimes you just want to write down something small, eureka should support this use-case as well.

**Comment by travv0** [comment]
I agree with this.  I'm trying it out, and after thinking of something I'll enter it at the prompt and then be presented with my editor, despite already having entered the thing I thought of.  (I'm also not really sure if there's a certain format I should be entering text into the readme in.  Just newline-separated by idea I guess?)

**Comment by simeg** [comment]
@tssund93 What you're doing when you're storing an idea is making a git commit. So the first step is the subject of the commit (what you'll see when you run `git log --oneline`) and how you structure your ideas in the README is up to you. I separate my ideas by a line of `-` characters.

But it's interesting that you bring it up - I assumed everyone thought about it like me but obviously not. The problem with not having your ideas in the README, but only in the git log, is that you would have to go to multiple places to see all of your ideas.

**Comment by travv0** [comment]
I was thinking it would be nice if there was a command that would take the commit message you enter and duplicate it in the readme, but obviously then there would have to be a specific way the readme is structured.  So doesn't really make sense, I suppose.

**Comment by tahervali** [comment]
So I really think, That the usecase where we enter new creates a new Read.md file will also cater to the use case we are discussing here.
@simeg @travv0 

What do you think?

I could take up the creating new read.md for every Idea first(Giving the user an option to choose whether to create a new read.md or use the old one)
